tests/test_monitor.py

```diff
@@ -227,6 +232,10 @@ def tick():
         next(time_counter)
         for counter in energy_counters.values():
             next(counter)
+        for i in range(len(monitor.cpu_indices)):
+            next(monitor.cpus._cpus[i].cpu_energy)
+            if i % 2 == 0:
+                next(monitor.cpus._cpus[i].dram_energy)
```

Comment:
Why is this only on even indices?

Reply:
This was to simulate having a DRAM sub package and not having one. I guess with only 2 CPUs, it's basically only when the index is 0.